src/panels/config/zha/zha-cluster-commands.ts

```diff
@@ -0,0 +1,275 @@
+              : ""
+          }
+        </paper-card>
+      </ha-config-section>
+    `;
+  }
+
+  private _renderCommandPicker(): TemplateResult {
+    return html`
+      <div class="command-picker">
+        <paper-dropdown-menu
+          label="Commands of the selected cluster"
+          dynamic-align=""
+          class="flex"
+        >
+          <paper-listbox
+            slot="dropdown-content"
+            selected="${this._selectedCommandIndex}"
+            @iron-select="${this._selectedCommandChanged}"
+          >
+            ${
+              this._commands.map(
+                (entry) => html`
+                  <paper-item
+                    >${entry.name + " (id: " + entry.id + ")"}</paper-item
+                  >
+                `
+              )
+            }
+          </paper-listbox>
+        </paper-dropdown-menu>
+      </div>
+
+      <div ?hidden="${!this._showHelp}" style="color: grey; padding: 16px">
+        Select a command to interact with
+      </div>
+    `;
+  }
+
+  private _renderCommandInteractions(): TemplateResult {
+    return html`
+      <div class="input-text">
+        <paper-input
+          label="Manufacturer code override"
+          type="number"
+          value="${this._manufacturerCodeOverride}"
+          @value-changed="${this._onManufacturerCodeOverrideChanged}"
+          placeholder="Value"
+        ></paper-input>
+      </div>
+      <div class="card-actions">
+        <ha-call-service-button
+          .hass="${this.hass}"
+          domain="zha"
+          service="issue_zigbee_cluster_command"
+          .serviceData="${this._issueClusterCommandServiceData}"
+          >Issue Zigbee Command</ha-call-service-button
+        >
+        <ha-service-description
+          .hass="${this.hass}"
+          domain="zha"
+          service="issue_zigbee_cluster_command"
+          ?hidden="${!this._showHelp}"
+        ></ha-service-description>
+      </div>
+    `;
+  }
+
+  private async _fetchCommandsForCluster(): Promise<any> {
+    if (this.selectedEntity && this.selectedCluster) {
+      this._commands = await this.hass!.callWS({
+        type: "zha/entities/clusters/commands",
+        entity_id: this.selectedEntity!.entity_id,
+        ieee: this.selectedEntity!.device_info!.identifiers[0][1],
+        cluster_id: this.selectedCluster!.id,
+        cluster_type: this.selectedCluster!.type,
+      });
+    }
+  }
+
+  private _computeIssueClusterCommandServiceData():
+    | IssueCommandServiceData
+    | undefined {
+    if (!this.selectedEntity || !this.selectedCluster) {
+      return;
+    }
+    return {
+      entity_id: this.selectedEntity!.entity_id,
+      cluster_id: this.selectedCluster!.id,
+      cluster_type: this.selectedCluster!.type,
+      command: this._commands[this._selectedCommandIndex].id,
+      command_type: this._commands[this._selectedCommandIndex].type,
+      /*
+      manufacturer: this._manufacturerCodeOverride
```

Comment:
Should this be here or removed?

Reply:
will be needed eventually but I haven't built the support in the back end yet. I can remove for now if necessary.